user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserOperator.java

```diff
@@ -96,7 +95,7 @@ public KafkaUserOperator(Vertx vertx,
     public Future<Set<NamespaceAndName>> allResourceNames(String namespace) {
         return CompositeFuture.join(super.allResourceNames(namespace),
                 invokeAsync(aclOperations::getUsersWithAcls),
-                invokeAsync(scramShaCredentialOperator::list)).map(compositeFuture -> {
+                invokeAsync(() -> scramShaCredentialOperator.list(new Reconciliation("list", KafkaUser.RESOURCE_KIND, namespace, "all-users")))).map(compositeFuture -> {
```

Comment:
Is the `list()` method used from multiple places? Or why fo we need to use the dummy reocnciliation here?